### PR TITLE
The logout symbol should be a little small and in line of username--Fixed Issue

### PR DESCRIPTION
**Pull request by suvarnakale**
The logout symbol should be a little small and in line of username--Fixed Issue

### CODE REVIEW

File: src/assets/i18n/global/hi-global.json [hunk 1]
```diff
@@ -64,7 +64,7 @@
   "MANDATORY":"अनिवार्य",
   "DESCRIPTION":"विवरण",
   "VISIBILITY": "दृश्यता",
-  "yes" : "हाँ",
+  "Yes" : "हाँ",
```
Comment (Pratikshakhandagale) [inline src/assets/i18n/global/hi-global.json:67]: @suvarnakale revert this changes

File: src/app/admin/publish/swaggerui/swaggerui.component.ts [hunk 2]
```diff
@@ -1,19 +1,48 @@
...
+            if (response.obj.access_token) {
+              console.log(response.obj.access_token);
+              const token = response.obj.access_token;
+              self.token;
+            }
+            return response;
+          },
+        requestInterceptor:
+          function (request) {
+            console.log('[Swagger] intercept try-it-out request');
```
Comment (Pratikshakhandagale) [inline src/app/admin/publish/swaggerui/swaggerui.component.ts:40]: @suvarnakale Remove console.log from all pages

Comment (suvarnakale) [reply src/app/admin/publish/swaggerui/swaggerui.component.ts:40]: yes
